### PR TITLE
Do not print executed commands in loop when checking deployments

### PR DESCRIPTION
**Pull request by mabartos**
We will prevent comprehensive logs when checking the Operator and related deployments. Sometimes, the important logs are hidden somewhere in the polluted logs :) 

@shawkins WDYT?

New logging would look like this:

```
error: the server doesn't have a resource type "keycloaks"
16:55:42 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:43 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:44 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:45 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:46 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:47 Waiting for Keycloak CRD
error: the server doesn't have a resource type "keycloaks"
16:55:48 Waiting for Keycloak CRD
```

Instead of this:

```
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
waiting for Keycloak CRD
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
waiting for Keycloak CRD
+ echo 'waiting for Keycloak CRD'
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
waiting for Keycloak CRD
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
+ (( c++ ))
+ (( c==max_retries ))
waiting for Keycloak CRD
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
+ (( c++ ))
+ (( c==max_retries ))
waiting for Keycloak CRD
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
+ kubectl get keycloaks
error: the server doesn't have a resource type "keycloaks"
+ echo 'waiting for Keycloak CRD'
waiting for Keycloak CRD
+ (( c++ ))
+ (( c==max_retries ))
+ sleep 1
```

### CODE REVIEW

**Review by shawkins — APPROVED** [review]
LGTM - thanks @mabartos, will go ahead and merge after an approval from @keycloak/maintainers

**Review by vmuzikar — COMMENTED** [review]
@mabartos Thanks for the PR!

File: .github/actions/conditional/conditions
Comment (mabartos) [reply .github/actions/conditional/conditions:25]: Yep, it's needed when some changes are made in the `operator/scripts`

Comment (vmuzikar) [reply .github/actions/conditional/conditions:25]: Good, point. Thanks for clarifying.

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**
